libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -65,8 +65,8 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
 
     // @Param: _USE
     // @DisplayName: Airspeed use
-    // @Description: use airspeed for flight control. When set to 0 airspeed sensor can be logged and displayed on a GCS but won't be used for flight. When set to 1 it will be logged and used. When set to 2 it will be only used when the throttle is zero, which can be useful in gliders with airspeed sensors behind a propeller
-    // @Values: 0:Don't Use,1:use,2:UseWhenZeroThrottle
+    // @Description: Enables airspeed use for automatic throttle modes and replaces control from THR_TRIM. Continues to display and log airspeed if set to 0. Uses airspeed for control if set to 1. Only uses airspeed when throttle = 0 if set to 2 (useful for gliders with airspeed sensors behind propellers).
```

Comment:
the use=2 description doesn't make sense

Reply:
What doesn’t make sense about it? You don’t want to use an airspeed sensor behind a prop, do you?